src/plugins/intel_gpu/tests/unit/test_cases/streams_test.cpp

```diff
@@ -77,32 +77,32 @@ class gpu_streams: public ::testing::Test {
 
         cldnn::network::ptr network0;
         cldnn::network::ptr network1;
-        if (is_caching_test) {
-            membuf mem_buf;
-            {
-                auto prog = program::build_program(engine, topology, get_test_default_config(engine));
-                {
-                    network0 = std::make_shared<cldnn::network>(prog, 0);
-                    std::ostream out_mem(&mem_buf);
-                    BinaryOutputBuffer ob = BinaryOutputBuffer(out_mem);
-                    network0->save(ob);
-                }
-            }
-            {
-                {
-                    std::istream in_mem(&mem_buf);
-                    BinaryInputBuffer ib = BinaryInputBuffer(in_mem, engine);
-                    auto pos = ib.tellg();
-                    network0 = std::make_shared<cldnn::network>(ib, get_test_stream_ptr(), engine, true, 0);
-                    ib.seekg(pos);
-                    network1 = std::make_shared<cldnn::network>(ib, get_test_stream_ptr(), engine, false, 0);
-                }
-            }
-        } else {
+        // if (is_caching_test) {
+        //     membuf mem_buf;
+        //     {
+        //         auto prog = program::build_program(engine, topology, get_test_default_config(engine));
+        //         {
+        //             network0 = std::make_shared<cldnn::network>(prog, 0);
+        //             std::ostream out_mem(&mem_buf);
+        //             BinaryOutputBuffer ob = BinaryOutputBuffer(out_mem);
+        //             network0->save(ob);
+        //         }
+        //     }
+        //     {
+        //         {
+        //             std::istream in_mem(&mem_buf);
+        //             BinaryInputBuffer ib = BinaryInputBuffer(in_mem, engine);
+        //             auto pos = ib.tellg();
+        //             network0 = std::make_shared<cldnn::network>(ib, get_test_stream_ptr(), engine, true, 0);
+        //             ib.seekg(pos);
+        //             network1 = std::make_shared<cldnn::network>(ib, get_test_stream_ptr(), engine, false, 0);
+        //         }
+        //     }
+        // } else {
```

Comment:
Will this test be updated w.r.t the latest change?

Reply:
I updated it. Thank you!